lib/core/logging-helper.ts

```diff
@@ -49,6 +49,16 @@ export class LoggingHelper {
         }
     }
 
+    public static prepareForFileLoggingAndDisableConsole(file: string): void {
+        LoggingHelper.logger.add(winston.transports.File,
```

Comment:
No, but we create the directory before creating the config file so we can register the log files even when the config is not created.
A missing scenario would be when we cannot even create the directory due permissions issues, but in that case we won't be able to create the log in that directory either.